hooks/command

```diff
@@ -2,6 +2,10 @@
 
 set -euo pipefail
 
+if [[ $OSTYPE == darwin* ]]; then
+  exec /bin/bash -c "$BUILDKITE_COMMAND"
+fi
```

Comment:
I was mostly thinking to rather check if we're on linux instead? with e.g `[[ $OSTYPE == "linux-gnu"* ]]` to make it clear that its not supported on anything else

Reply:
Is that safe to do for all Linux-ish operating systems this supports? I'm not sure if BSD or something would match that or not?

Reply:
Yeah true let's leave it like this I'd say then!